### PR TITLE
RadioGroup: add some event tests

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.editors/radioGroup.tests.js [hunk 1]
```diff
@@ -341,6 +356,19 @@ module('value', moduleConfig, () => {
...
+        const handler = sinon.spy();
+        const $radioGroup = createRadioGroup({
+            items: [1, 2, 3]
+        });
+        const radioGroup = getInstance($radioGroup);
+
+        radioGroup.on('valueChanged', handler);
+        $(radioGroup.itemElements()).first().trigger('dxclick');
+
+        assert.ok(handler.calledOnce, 'value changed');
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets.editors/radioGroup.tests.js]: Check args of the `valueChanged` event. `handler.lastCall.args[0]`

File: testing/tests/DevExpress.ui.widgets.editors/radioGroup.tests.js [hunk 2]
```diff
@@ -135,6 +135,21 @@ module('buttons group rendering', () => {
...
+    test('onContentReady - subscription using "on" method', function(assert) {
+        const onContentReadyHandler = sinon.stub();
+
+        const instance = getInstance(
+            createRadioGroup({
+                dataSource: ['str1', 'str2', 'str3']
+            })
+        );
+
+        instance.on('contentReady', onContentReadyHandler);
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets.editors/radioGroup.tests.js:153]: I think it's worth checking that the radio buttons are rendered at the time of the event (check the markup in the handler).

Comment (San4es) [reply testing/tests/DevExpress.ui.widgets.editors/radioGroup.tests.js:153]: @DokaRus @ksercs 
So, do we need to make this test async?

Comment (DokaRus) [reply testing/tests/DevExpress.ui.widgets.editors/radioGroup.tests.js:153]: Yep. We can add `expect` assertion and resolve async promise on the `contentReady` event

Comment (ksercs) [reply testing/tests/DevExpress.ui.widgets.editors/radioGroup.tests.js:153]: Is it ok now?

**Review by DokaRus — DISMISSED** [review]

File: testing/tests/DevExpress.ui.widgets.editors/radioGroup.tests.js [hunk 3]
```diff
@@ -341,6 +360,25 @@ module('value', moduleConfig, () => {
...
+        });
+        const radioGroup = getInstance($radioGroup);
+
+        radioGroup.on('valueChanged', handler);
+        $(radioGroup.itemElements()).first().trigger('dxclick');
+
+        const e = handler.lastCall.args[0];
+
+        assert.ok(handler.calledOnce, 'handler was called');
+        assert.strictEqual(e.component, radioGroup, 'component is correct');
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets.editors/radioGroup.tests.js:376]: You could make these asserts directly inside the 'valueChanged' handler, but OK

File: testing/tests/DevExpress.ui.widgets.editors/radioGroup.tests.js [hunk 4]
```diff
@@ -135,6 +135,25 @@ module('buttons group rendering', () => {
         instance.option('dataSource', [1, 2, 3]);
         assert.strictEqual(onContentReadyHandler.callCount, 2);
     });
+
+    test('onContentReady - subscription using "on" method', function(assert) {
+        const done = assert.async(1);
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets.editors/radioGroup.tests.js]:
```suggestion
        const done = assert.async();
```
1 is a default value.

**Review by San4es — DISMISSED** [review]

**Review by DokaRus — APPROVED** [review]

**Pull request merged**